README.md

```diff
@@ -20,6 +20,9 @@ schema file - `./source_data_schema.json`.
 ## Guide & Tips:
  
 - The team is interested in the approach you follow for solving this problem
+- Approach this problem as you would to build an end to end ETL pipeline.
```

Comment:
What does end-to-end mean? Maybe paint a better picture of the problem space so that people can get a glimpse into the sort of problems this type of etl might face.